Changes

```diff
@@ -8,6 +8,34 @@ The changes in each SiliconCompiler release version are described below. Commit
 version shown in (). Where applicable, the contributors that suggested a given
 feature are shown in [].
 
+SiliconCompiler 0.10.0 (2023-03-08)
+=========================================
+
+**Major:**
+
+* Schema overhauled (see `schema documentation <https://docs.siliconcompiler.com/en/stable/reference_manual/schema.html>`_ for details):
+
+  * added ability to override certain schema parameters on a per-step/index basis.
```

Comment:
Do we want to capitalize the first letter of these sub-bullets?

Reply:
yes, to be consistent